schema/codelists/addressType.csv

```diff
@@ -1,7 +1,8 @@
-code,title,description,technical note
-placeOfBirth,Place of birth
-home,Home address
-residence,Residential address
-registered,Registered address
-service,Service address
-alternative,Alternative address
\ No newline at end of file
+code,title,description,technical note,
+placeOfBirth,Place of birth,The place where a person was born. This must only be used to describe natural persons. Only a town and country are required.,,
+home,Home address,The address of the house or apartment which is someone's main place of residence. This must only be used for natural persons.,,
```

Comment:
I think remove this and merge with residence.

Reply:
"Home address,The address <s>of the house or apartment</s> which is someone's main place of residence. This must only be used for natural persons"

CRS lets you use a postbox if it has physical location.

Reply:
I can't square the description of this being a main place of residence with the possibility that it might be a postbox, but I'm happy to lose the "of the house or apartment" as other dwelling types can also be places of residence

Reply:
Edited as per suggestion